cmd/promtool/main.go

```diff
@@ -99,6 +89,30 @@ func checkConfig(t cli.Term, filename string) ([]string, error) {
 		ruleFiles = append(ruleFiles, rfs...)
 	}
 
+	exists := func(fn string) bool {
+		// Nothing set, nothing to error on.
+		if fn == "" {
+			return true
+		}
+		_, err := os.Stat(fn)
+		return !os.IsNotExist(err)
```

Comment:
So if accessing the filesystem fails for whatever reason, we treat this as if the file exists?